mock-hpos-api/defaultResponse.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash')
+
+// mock of normal hpos api responses
+
+const happs = [{
+  id: 1,
+  name: 'HoloFuel'
+}, {
+  id: 2,
+  name: 'Community'
+}, {
+  id: 3,
+  name: 'Elemental Chat'
+}]
+
+
+const data = {
+  get: {   
+    '/hosted_happs': {
+      hosted_happs: happs
+    },
```

Comment:
I thought this was now deprecated in favour of getting it from HHA via `hpos-holochain-api`?

Reply:
Yes, that's right. Will update